controllers/events/event.go

```diff
@@ -314,3 +317,20 @@ func InitFunction(image string) *ofcore.Function {
 
 	return function
 }
+
+func getDefaultConfig(ctx context.Context, c client.Client, log logr.Logger) map[string]string {
```

Comment:
https://github.com/OpenFunction/OpenFunction/blob/main/controllers/core/serving_controller.go#L313
Looks like a bit of a duplicate definition, maybe it should be moved to `pkg/util`